pkg/pillar/cmd/zedagent/handlepatchenvelopes.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package zedagent
+
+import (
+	"bytes"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/lf-edge/eve-api/go/info"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"google.golang.org/protobuf/proto"
+)
+
+func composePatchEnvelopeUsage(appUUID string, ctx *zedagentContext) []*info.ZInfoPatchEnvelopeUsage {
+	result := []*info.ZInfoPatchEnvelopeUsage{}
+
+	for _, c := range ctx.subPatchEnvelopeUsage.GetAll() {
+		peUsage := c.(types.PatchEnvelopeUsage)
+		if peUsage.AppUUID == appUUID {
+			result = append(result, &info.ZInfoPatchEnvelopeUsage{
+				Uuid:              peUsage.PatchID,
```

Comment:
Is there some Uuid vs. UUID issue here which makes yetus complain?

Reply:
This yetus complaints should go after merging correct [update](https://github.com/lf-edge/eve/pull/3566) for eve-api/go and rebasing this PR to master